style.css

```diff
@@ -0,0 +1,252 @@
+body {
+    display: flex;
+    position: relative;
+    width: 1440px;
+    height: 1000px;
+    background-color: #141b1c;
+}
+.section_1 {
+    position: absolute;
+    left: 132px;
+    right: 928px;
+    top: 115px;
+    bottom: 115px;
+    width: 380px;
+    height: 770px;
+    background-color: #334247;
+    box-shadow: 0px 6px 6px rgba(0, 0, 0, 0.087466);
+}
+.section_2 {
+    position: absolute;
+    left: 530px;
+    right: 530px;
+    top: 115px;
+    bottom: 115px;
+    background-color: #334247;
+    width: 380px;
+    height: 770px;
+    overflow: hidden;
+}
+.Popular-background {
+    position: absolute;
+    width: 25.01px;
+    height: 147.08px;
+    background-color: #dc3f45;
+    position: absolute;
+    top: -38.41px;
+    right: 21.69px;
+    bottom: 658.91px;
+    left: 332px;
+    transform: rotate(-45deg);
+    
+}
+.Popular {
+    position: absolute;
+    width: 75.76px;
+    height: 17px;
+    left: -23.93px;
+    right: 6.02px;
+    top: 67.31px;
+    bottom: 28.79px;
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 17px;
+    color: #ffffff;
+    transform: rotate(90deg);
+    text-transform: uppercase;
+}
+.section_3 {
+    position: absolute;
+    left: 928px;
+    right: 132px;
+    top: 112px;
+    bottom: 118px;
+    background-color: #334247;
+    width: 380px;
+    height: 770px;
+}
+h1 {
+    position: absolute;
+    color: #ffffff;
+    font-size: 35px;
+    text-align: center;
+    font-family: 'Montserrat';
```

Comment:
use `font-family` in `body` if you have 1 font for all page

Reply:
the code uses 2 types of fonts